### DISCUSSION TITLE
Geometry of work zones: What should it represent?

### DISCUSSION THREAD

**Discussion by wesleyalforddot** [usdot-jpo-ode/jpo-wzdx#134] [Q&A]
The following is a question the technical assistance subgroup received from Maricopa County DOT about GeoJSON representation of a work zone. 

### **Please weigh in on how you would represent the work zone and what would be most useful for data consumers.**

Here is what the work zone plan looks like.  Notice the locations of the construction cones shown in the image.
 
Plan:
 ![plan](https://user-images.githubusercontent.com/31485082/94287635-8dbb7680-ff24-11ea-8e24-7ec45f3f3211.jpg)

 
We initially defined the GeoJSON to show where the cones are to be placed using sub-identifiers to tie the various line segments together.:
 
Option 1:
![option 1](https://user-images.githubusercontent.com/31485082/94287668-96ac4800-ff24-11ea-9976-8ecde35295af.png)

 
 
But upon further consideration, we believe that perhaps the line should represent the lane in which the vehicle is allowed to go:
 
Option 2:
 
 ![option 2](https://user-images.githubusercontent.com/31485082/94287692-9e6bec80-ff24-11ea-927d-90219bda32b3.jpg)

We even considered using points to show where the cones are:
 
Option 3:
 ![option 3](https://user-images.githubusercontent.com/31485082/94287734-a75cbe00-ff24-11ea-8425-2ceea3c0fcb3.png)
 
In which way should be represent the work zone geometry?  Other option?

**Comment by sknick-iastate** [comment]
For reference, a corresponding issue and discussion is also available in #78 that discusses how a work zone should be broken into multiple road events.
